Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-runtime
+FROM ubuntu:16.04
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
-ENV PATH /opt/conda/bin:$PATH
+# Add nvidia and conda binaries to path.
+ENV PATH /usr/local/nvidia/bin/:/opt/conda/bin:$PATH
 ENV PYTHONHASHSEED 2157
 
-# Override Nvidia's default LD paths, since they're misconfigured in the base image.
-ENV LD_LIBRARY_PATH /usr/local/cuda-8.0/lib64:$LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/local/nvidia/lib64:$LD_LIBRARY_PATH
```

Comment:
Here LD_LIBRARY_PATH is likely empty, and pre-pending to empty library path is a bad practice, you might end up inadvertently adding your working dir to your $LD_LIBRARY_PATH, see https://joshumax.github.io/general/2017/06/08/how-torch-broke-ls.html or https://patchwork.ozlabs.org/patch/413215/

Reply:
Thanks a lot for reporting originally and this review 👍

Reply:
It is empty.  I will remove the reference to `$LD_LIBRARY_PATH`.  Thanks!